fastlane/Fastfile

```diff
@@ -22,7 +22,7 @@ private_lane :merged_prs_since_last_release do |options|
     server_url: "https://api.github.com",
     api_token: github_token,
     http_method: "GET",
-    path: "/repos/guardian/#{repo}/pulls?state=closed&base=master&sort=closed&direction=desc"
+    path: "/repos/guardian/#{repo}/pulls?state=closed&base=master&sort=created&direction=desc&per_page=100"
```

Comment:
https://developer.github.com/v3/guides/traversing-with-pagination/#changing-the-number-of-items-received